### DISCUSSION TITLE
Integrate Kolors Models

### DISCUSSION THREAD

**Discussion by thiner** [lllyasviel/Fooocus#3366] [Ideas]
Kolors models are showing impressive Chinese character painting capabilities. Is it possible to integrate this model family in Fooocus? 
https://huggingface.co/Kwai-Kolors

![](https://huggingface.co/Kwai-Kolors/Kolors-Inpainting/resolve/main/imgs/Comparison_Results.png)

**Comment by mashb1t** [comment]
Not for now, see https://github.com/lllyasviel/Fooocus/issues/3248#issuecomment-2213298584
